app/lib/rendering/OpenTypePen.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2015, Felipe Correa da Silva Sanches <juca@members.fsf.org>
+ * 
+ * You should have received a copy of the MIT License along with this program.
+ * If not, see http://www.opensource.org/licenses/mit-license.php
+ * 
+ */
+define(
+    [
+        'opentype'
+      , 'ufojs/tools/pens/BasePen'
+    ],
+    function(
+        opentype
+      , BasePen
+) {
+    "use strict";
+
+    /*constructor*/
+    function OpenTypePen () {
```

Comment:
As an argument, the constructor `BasePen` takes a glyphSet, to make it possible to draw components directly into the path. When you don't do this, you need to implement `addComponent` https://github.com/graphicore/ufoJS/blob/master/lib/tools/pens/BasePen.js#L100 yourself.
That is also exactly what I want to see here, if components are supported by opentype.js. If not, the glypset argument is mandatory.

Also, important, call the parent constructor right here:
```js
Parent.call(this, undefined);
```

This is described here: https://github.com/metapolator/metapolator/wiki/coding-style#lets-do-inheritance